c172-sound.xml

```diff
@@ -289,6 +292,80 @@
    </condition>
   </kap140beep>
 
+  <left-pontoon>
+   <name>left-pontoon</name>
+   <mode>looped</mode>
+   <path>Sounds/water-fast.wav</path>
+   <condition>
+    <greater-than>
+     <property alias="/params/spray_wake_speed_kt/property"/>
+     <value>5</value>
+    </greater-than>
+   </condition>
+   <position>
+    <x>0.0</x>
+    <y>1.1</y>
+    <z>-2.0</z>
+   </position>
+   <reference-dist>15.0</reference-dist>
+   <max-dist>50.0</max-dist>
+   <volume>
+    <property alias="/params/spray_wake_speed_kt/property"/>
+    <factor>0.04</factor>
+    <offset>0.1</offset>
+    <min>0.1</min>
+    <max>2.0</max>
+   </volume>
+  </left-pontoon>
+
+  <right-pontoon>
+   <name>right-pontoon</name>
+   <mode>looped</mode>
+   <path>Sounds/water-fast.wav</path>
+   <condition>
+    <greater-than>
+     <property alias="/params/spray_wake_speed_kt/property"/>
+     <value>5</value>
+    </greater-than>
+   </condition>
+   <position>
+    <x>0.0</x>
+    <y>-1.1</y>
+    <z>-2.0</z>
+   </position>
+   <reference-dist>15.0</reference-dist>
+   <max-dist>50.0</max-dist>
+   <volume>
+    <property alias="/params/spray_wake_speed_kt/property"/>
+    <factor>0.04</factor>
+    <offset>0.1</offset>
+    <min>0.0</min>
+    <max>2.0</max>
+   </volume>
+  </right-pontoon>
+
+  <pontoon-in-water>
+   <name>pontoon-in-water</name>
+   <mode>looped</mode>
+   <path>Sounds/water-still.wav</path>
+   <condition>
+    <greater-than>
+     <property alias="/params/spray_wake_speed_kt/property"/>
+     <value>0</value>
+    </greater-than>
+   </condition>
+   <position>
+    <x>0.0</x>
+    <y>0.0</y>
+    <z>-2.0</z>
+   </position>
+   <reference-dist>10.0</reference-dist>
+   <max-dist>25.0</max-dist>
```

Comment:
Also make sure you lower the volume if `/sim/current-view/internal` is true. You could use `<volume>` with a `<property` for this, for example.